bedrock/cms/management/commands/download_media_to_local.py

```diff
@@ -0,0 +1,78 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import os
+import sys
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from google.cloud import storage
+
+from bedrock.cms.models import BedrockImage
+from bedrock.settings.base import path as build_path
+
+BUCKETS = {
+    "dev": "bedrock-nonprod-dev-cms-media",
+    "stage": "bedrock-nonprod-stage-cms-media",
+    "prod": "bedrock-prod-prod-cms-media",
+}
+
+
+class Command(BaseCommand):
+    help = """Downloads public media files from the appropriate cloud bucket
+    so that they match the sqlite database being currently used."""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--environment",
+            default="dev",
+            help="Which Bedrock environment are you downloading from? Values are dev | stage | prod (default is dev)",
+        )
+        parser.add_argument(
+            "--redownload",
+            action="store_true",
+            dest="redownload",
+            default=False,
+            help="If passed, will force re-download of all the assets in the relevant bucket.",
+        )
+
+    def handle(self, *args, **options):
+        # If we're not using sqlite, stop, because this tool isn't for that
+
+        storage_client = storage.Client.create_anonymous_client()
+
+        if settings.DATABASES["default"]["ENGINE"] != "django.db.backends.sqlite3":
+            self.stderr.write(f"This command only works if you are using sqlite as your local DB. Got {settings.DATABASES['default']['engine']}\n")
```

Comment:
The `if` uses `"ENGINE"` but the string uses `'engine'`. I'm thinking this would be case-sensitive. Also, while I'm here, the above setting of `storage_client` could go below this check.